cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
+
+    func callbacks() {
+        view.backgroundColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 1)
+        customView.didTapButton = {
+            if let payment = self.payment {
+                self.delegate.didFinishFlow(amount: payment.amount
+                                            currency: payment.currency)
+            }
+        }
+    }
+
+    func viewDidLoad() {
+        navigationController?.setNavigationBarHidden(false, animated: false)
+        customView.centerXAnchor.constraint(equalTo: view.centerXAnchor)
+        customView.centerYAnchor.constraint(equalTo: view.centerYAnchor)
+        view.addSubview(customView)
```

Comment:
1. separate `22`, `23` and `24` lines to a separate method called `addCustomPaymentView()`
2. call view.addSubView(...) before installing constraints as it will crash as the view is not in the hierarchy.
3. call customView.translatesAutoresizingMaskIntoConstraints = false to add the constraints yourself.
4. add constraints using NSLayoutConstraint.activate([ `add constraints here` ]) or add .active = true for every constraint to be active
5. you need height and width for the customView as it is missing.